src/NBitcoin/ChainBase.cs

```diff
@@ -153,6 +200,11 @@ public ChainedBlock FindFork(IEnumerable<uint256> hashes)
             return null;
         }
 
+        /// <summary>
```

Comment:
there is another version of FindFork untouched by this PR, but its comment makes no sense and need rephrasing, its summary reads as `Returns the first found block.` which is confusing

Reply:
I reworked this comment.